include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -547,13 +547,14 @@ struct __parallel_copy_if_static_single_group_submitter<_Size, _ElemsPerItem, _W
                     }
                 });
         });
-        return __future(__event, __result);
+
+        return {std::move(__event), std::move(__result)};
```

Comment:
Same comment as before.  To me it makes sense to include `__future` here for clarity.

Reply:
Fixed